SourceX/miniwin/misc_msg.cpp

```diff
@@ -17,7 +17,11 @@ static int translate_sdl_key(SDL_Keysym key)
 {
 	// ref: https://wiki.libsdl.org/SDL_Keycode
 	// ref: https://docs.microsoft.com/en-us/windows/win32/inputdev/virtual-key-codes
-	int sym = key.sym;
+#ifdef USE_SDL1
+	SDLKey sym = key.sym;
```

Comment:
You could implement `SDL_Keycode` as `typedef SDLKey` in sdl2_to_1_2_backports.h

Reply:
Done